nuxt/api/tracks.ts

```diff
@@ -0,0 +1,86 @@
+import { NuxtAxiosInstance } from '@nuxtjs/axios';
+import {
+  PersistedEntity,
+  TimestampedEntity,
+  createParams,
+  useIncludes,
+  PaginatedResponse,
+  usePagination,
+  PaginationOptions,
+} from '@/api/common';
+import { Reciter } from '@/api/reciters';
+import { Album } from '@/api/albums';
+
+/*
+ * Entity Definitions
+ */
+export interface Track extends PersistedEntity, TimestampedEntity {
+  reciterId: string;
+  reciter?: Reciter;
+  albumId: string;
+  album?: Album;
+  title: string;
+  slug: string;
+  audio: string;
+  lyrics: string;
```

Comment:
Slight nitpick: can we move optional attributes to the bottom of the interface declaration?